lib/outbox/src/main/kotlin/no/nav/amt/lib/outbox/OutboxService.kt

```diff
@@ -0,0 +1,59 @@
+package no.nav.amt.lib.outbox
+
+import no.nav.amt.lib.utils.objectMapper
+import java.util.UUID
+
+/**
+ * Provides a high-level API for interacting with the outbox.
+ * This service simplifies the creation and management of outbox events,
+ * abstracting away the underlying repository details.
+ */
+class OutboxService {
+    private val repository = OutboxRepository()
+
+    /**
+     * Creates a new outbox event and persists it to the database.
+     *
+     * @param T The type of the value being sent.
+     * @param key The key of the event, typically a unique identifier for the entity.
+     * @param value The value (payload) of the event.
+     * @param topic The Kafka topic to which the event will be published.
+     * @return The created [OutboxEvent].
+     */
+    fun <T : Any> newEvent(
```

Comment:
For å pirke litt. Syns det kunne vært litt mer logisk om denne het "save()" fordi newEvent høres ut som noe som automatisk trigger noe, potensielt noe som legges på kafka, eller annen magi